packages/vuelidate/src/index.js

```diff
@@ -64,12 +65,14 @@ export const VuelidateMixin = {
       : options.validations
 
     if (!options.computed) options.computed = {}
-    if (options.computed.$v) return
+    if (options.computed.v$) return
```

Comment:
is `$v` still not allowed? Didnt Evan change it?
I think this will very much confuse people :/

Reply:
It didn’t work for me when I tried. :(

Reply:
This will cause much confusion... not to mention backwards compatibility. A search and replace is easy, but still.

Reply:
How do Vuex and VueRouter do it now?

Reply:
`$v` works when added using old Options API! So it would only affect people that want to use the new `setup` approach.

Reply:
That doesnt sound great... Its going to lead to allot of confusion.